app/helpers/course/statistics_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Course::StatisticsHelper
+  # Convert time in seconds to HH:MM:SS format.
+  def seconds_to_str(total_seconds)
```

Comment:
Not sure if this will be useful to be placed in `ApplicationFormattersHelper` (or a sub-helper under this one) instead? Could be reused in other parts of the application.